Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-08-12
Forwarded: 2012-08-21
Description: fixes FTBFS with format hardening and '--with-gtk2'
 Workaround for error: "format not a string literal and
 "no format arguments [-Werror=format-security]"

--- a/goffice-bits2/goffice/gtk/goffice-gtk.c
+++ b/goffice-bits2/goffice/gtk/goffice-gtk.c
@@ -853,7 +853,7 @@
 			 GTK_DIALOG_DESTROY_WITH_PARENT,
 			 GTK_MESSAGE_WARNING,
 			 GTK_BUTTONS_OK_CANCEL,
-			 msg);
+			 "%s", msg);
 		gtk_dialog_set_default_response (GTK_DIALOG (dialog),
 			overwrite_by_default ? GTK_RESPONSE_OK : GTK_RESPONSE_CANCEL);
 		result = GTK_RESPONSE_OK ==
